cmd/minishift/cmd/oc_env.go

```diff
@@ -104,14 +106,7 @@ var ocEnvCmd = &cobra.Command{
 		util.ExitIfNotRunning(host.Driver, constants.MachineName)
 
 		var shellCfg *OcShellConfig
-		var ocPath string
-		if constants.ProfileName != profileActions.GetActiveProfile() {
-			// When PROFILE_NAME is not an active profile i.e. oc-env --profile PROFILE_NAME
-			// is used we need to findout the oc path
-			ocPath = getOcPath()
-		} else {
-			ocPath = config.InstanceConfig.OcPath
-		}
+		ocPath := config.InstanceConfig.OcPath
```

Comment:
👍  It always give the correct ocPath because in root.go we initialize this every time.